src/main/java/net/education/kyivstar/config/SchemaAndTableCreator.java

```diff
@@ -0,0 +1,67 @@
+package net.education.kyivstar.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.DATABASE_PATH;
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.TABLES_PATH;
+import static net.education.kyivstar.services.util.Utils.readSqlScriptFromFileStream;
+
+public class SchemaAndTableCreator {
+    private static final Logger logger = LoggerFactory.getLogger(SchemaAndTableCreator.class);
+    private final EmbeddedMariaDbLifeCycle embeddedMariaDbLifeCycle;
+
+    public SchemaAndTableCreator(EmbeddedMariaDbLifeCycle embeddedMariaDbLifeCycle) {
+        this.embeddedMariaDbLifeCycle = embeddedMariaDbLifeCycle;
+    }
+
+    public void createDataBase() {
+        try (var connDb = embeddedMariaDbLifeCycle.connectMariaDb(false);
+             var statement = connDb.createStatement()) {
+            logger.info("Data base creation is initialized");
+            connDb.setAutoCommit(false);
+
+            var inputStream = getClass().getClassLoader().getResourceAsStream(DATABASE_PATH);
```

Comment:
It's good practice to explicitly close resources to prevent potential resource leaks, especially in long-running applications.

Reply:
+